### PR TITLE
EP-705 iOS - Make a server payment

### PR DESCRIPTION
**Pull request by AlexeiJovmir**
singletone helper for nsdateformat
add server to server payment

### CODE REVIEW

File: Source/Helpers/NSDateFormator/JPFormatters.m [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//
+//  JPFormatters.m
+//  JudoKitObjC
+//
+//  Copyright (c) 2019 Alternative Payments Ltd
```
Comment (stpopa) [inline Source/Helpers/NSDateFormator/JPFormatters.m:5]: `//  Copyright (c) 2020 Alternative Payments Ltd` :)

File: Source/Helpers/NSDateFormator/JPFormatters.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//
+//  JPFormatters.h
+//  JudoKitObjC
+//
+//  Copyright (c) 2019 Alternative Payments Ltd
```
Comment (stpopa) [inline Source/Helpers/NSDateFormator/JPFormatters.h:5]: `//  Copyright (c) 2020 Alternative Payments Ltd`

File: Source/Helpers/NSDateFormator/JPFormatters.m (hunk 1 above)
Comment (AlexeiJovmir) [reply Source/Helpers/NSDateFormator/JPFormatters.m:5]: copy paste copy paste)

**Comment by stpopa** [comment]
@AlexeiJovmir can we fix the code smells please?

**Pull request merged**